### ISSUE TITLE
Where are the Vim script communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/vimscript#61]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Vim script supported by one or more large organizations?
* Does Vim script have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Vim script? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Vim script? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Vim script? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Vim script?
* Are there popular newsletters for Vim script?
* Is Vim script taught at programming bootcamps? (If so, what are some examples?)
* Is Vim script taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Vim script?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by mhinz** [comment]
> Is Vim script supported by one or more large organizations?

Hard to say. It's part of Vim and it's not exactly clear if there is any company who actually supports it financially. No company is contributing code as far as I know. All patches come from Vim enthusiasts.

> Does Vim script have an official community manager?

Nope. Only a single person controls Vim (one reason why it was forked as Neovim), [Bram Moolenaar](https://en.wikipedia.org/wiki/Bram_Moolenaar). He works at Google, but maintains Vim in  his spare time.

> Do you know of specific communities (online or offline) that are enthusiastic about Vim script? (Chat communities, forums, meetups, student clubs, etc)

Biggest communities are probably [r/vim](https://www.reddit.com/r/vim/) and #vim on Freenode.

> Are there popular conferences for Vim script?

A few Vim conferences exist. Most notably [VimConf](https://vimconf.vim-jp.org/) and [VimFest](https://vimfest.org/).

> Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Vim script? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

I doubt that. VimL is part of Vim and Vim is _mostly_ used by programmers who enjoy text editors (although there are popular GUIs, of course).

> Are there specific groups or programs dedicated to mentoring people in Vim script?

I doubt anyone mentors for free. The only paid program I know is http://vimcasts.org/training, but then I never looked for others.

> Are there popular newsletters for Vim script?

I always wanted to start one, but wasn't sure if it's worth the effort. ;-) There were a few newsletters over the years, but most quit as far as I know.

> Is Vim script taught at programming bootcamps?

Not that I heard of.

> Is Vim script taught at universities? 

I highly doubt that.

> In other words: where do people care a lot and/or know a lot about Vim script?

I would probably try looking for mentors at r/vim, #vim on Freenode, or directly message authors of popular and maintained plugins.

**Comment by kytrinyx** [comment]
Thanks so much, this is great.

> > Is Vim script taught at universities?
> I highly doubt that.

😂

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼
